src/azure-cli/install.sh

```diff
@@ -10,6 +10,7 @@
 set -e
 
 AZ_VERSION=${VERSION:-"latest"}
+AZ_DEV_VERSION=${VERSION:-"latest"}
```

Comment:
The right hand side of this will need to match the name of the option (uppercased). If the option name is `azureDevCliVersion`, the environment variable that will be set will be `AZUREDEVCLIVERSION`, so you could have the following
```suggestion
AZ_DEV_VERSION=${AZUREDEVCLIVERSION:-"latest"}
```